README.md

```diff
@@ -29,7 +29,7 @@ The output will list all installed templates (not listed for brevity). In that l
 Template Name                                         Short Name                       Language    Tags
 ----------------------------------------------------  -------------------------------  ----------  ------------------------------------------
 ...
-Amido Stacks Web Api CQRS - Add to existing solution  stacks-add-web-api-cqrs          [C#]        Stacks/WebAPI/CQRS/api
+Amido Stacks Web API                                  stacks-add-web-api-cqrs          [C#]        Stacks/WebAPI/CQRS/api
```

Comment:
Isn't this `Amido Stacks CQRS Projects` now?

Reply:
Changed name of console output to reflect template name